app/(user)/campaign-application/[...IdStep]/components/ApplicationStep1.js

```diff
@@ -0,0 +1,153 @@
+'use client';
```

Comment:
these files should not be under "user"
A guest can fill them too, and they are more under (candidate)